core/services/pipeline/task.bridge.go

```diff
@@ -68,6 +71,7 @@ func (t *BridgeTask) Run(ctx context.Context, vars Vars, inputs []Result) Result
 		)
 	}
 
+	// TODO: pass t.id on the meta key? or where?
```

Comment:
It actually doesn't need to be passed anywhere, the important thing is to send a `responseURL`.

See spec: https://docs.chain.link/docs/developers/